src/patterns/dosdp-patterns/cellBearerOfQuality.yaml

```diff
@@ -0,0 +1,50 @@
+pattern_name: cellBearerOfQuality
+pattern_iri: http://purl.obolibrary.org/obo/cl/cellBearerOfQuality
+
+description: A cell that has a specific quality, such as binucleate.
+examples: CL_0000413 haploid cell, CL_0000225 anucleate cell, CL_0000227 binucleate cell (25 total)
```

Comment:
This makes it seem like the pattern can only be used with generic cell. Add some examples for specific cells